project/dexter-cs/DexterCRC/Src/CheckerLogic/WithBrace.cs

```diff
@@ -45,8 +45,8 @@ public WithBrace()
         public bool HasDefect(object value)
         {
             string statement = value.ToString();
-            return !statement.StartsWith("{", StringComparison.Ordinal)
-                    && !statement.EndsWith("}", StringComparison.Ordinal);
+            return !(statement.StartsWith("{", StringComparison.Ordinal)
```

Comment:
Previously it was a conjunction of negations. Now it is a negation of conjunctions. These are not the same. Are you sure?

Reply:
After consulting Mr. Augustus De Morgan, I am pretty sure it should correct. Let me explain.

Previously, if there was only one bracket (either openning or closing) the function `HasDefect()` returned `false`, which meant that there was no defect and everything was correct. This was obviously false, because bracing requires both the opening and the closing brace. 
Now, the `HasDefect()` function is sensitive for the presence of both of the braces and returns `false` (=no defect) if only if there are both braces.

Reply:
Ok, sounds convincing